pgml-extension/src/api.rs

```diff
@@ -803,7 +803,7 @@ fn tune(
     project_name: &str,
     task: default!(Option<&str>, "NULL"),
     relation_name: default!(Option<&str>, "NULL"),
-    y_column_name: default!(Option<&str>, "NULL"),
+    _y_column_name: default!(Option<&str>, "NULL"),
```

Comment:
Why the underscore? Is it because it's not used?

Reply:
That's correct.